tests/collapse.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { test, expect } from '@playwright/test';
+
+test('Collapse E2E Tests', async ({ page }) => {
+  await page.goto('/test.html');
+
+  const collapse = page.getByTestId('collapse')
+
+  // Main title should display
+  const collapseTitleText = 'Robots...'
+  const collapseTitle = collapse.getByText(collapseTitleText)
+  await expect(collapseTitle).toBeVisible()
+  await expect(collapseTitle).toHaveText(collapseTitleText)
+
+  // Slot:title should display
+  const titleSlotText = '(title)'
+  const titleSlot = collapse.getByText(titleSlotText)
+  await expect(titleSlot).toBeVisible()
+  await expect(titleSlot).toHaveText(titleSlotText)
```

Comment:
this test does not check for relative position of the title slot ("the collapse should render the child element  after the "title" attribute value in the collapse title")